spec/api-spec.json

```diff
@@ -0,0 +1,111 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "Sic-Code-API",
+    "version": "1.0"
+  },
+  "servers": [
+    {
+      "url": "http://api.chs.local/siccode-api",
```

Comment:
I think that in the meeting with Steve, we talked about this becoming /sic-code-api

Reply:
updated to /sic-code-search as per other services